packages/acceptance-tests/pkg-tests-specs/sources/node-modules.test.ts

```diff
@@ -1722,4 +1722,31 @@ describe(`Node_Modules`, () => {
         });
       }),
   );
+
+  it(`should support supportedArchitectures`,
+    makeTemporaryEnv(
+      {
+        dependencies: {
+          native: `1.0.0`,
+        },
+      },
+      async ({path, run}) => {
+        await xfs.writeJsonPromise(ppath.join(path, Filename.rc), {
```

Comment:
nitpick - you can remove the file write here and instead move .yarnrc.yml config object into the second argument of `makeTemporaryEnv` above

Reply:
It doesn't support objects/nested configurations so I can't do that.